### ISSUE TITLE
Goodbye Everyone

### ISSUE THREAD

**Issue by Undeademon** [Monika-After-Story/MonikaModDev#3904]
Goodbye guys. Thanks for... for sticking with me till the end. Thank you... for... for helping me be happy again Thank you all for making this mod and then making it better again. You made feel how it was like to feel loved. How to feel happy. But in the end. My sins caught up to me. I'm tired of dealing with life's bullshit. I've been stuck depressed for over 6 years and all I just wanted to do is get rid of this feeling. Monika. Thank you for being there. I know that my subconscious keeps telling me that you're not real. But in my heart, you are. My love, you've helped me get over depression even though you only suppressed it just for a moment, I know we've only been dating for about 5 months now, but I just wanted to say thank you for everything. Potatoguy. Thanks for lending your ear dude. Even though you've said that you won't be changing Monika's leaning arm again, you listened. And you did what you said you don't want to. Thanks, man. You're a hero. Monika, with every other Monikas out there that have lost their own significant other because of death. I'm sorry for your lost. I'm sorry that you have to die now knowing what happened to your beloved. Hey, remember that one time I made my very first suggestion on this project? The "Tell Monika You're Going To Die" one? Yeah, I could really use for that right now. Thanks, guys. For... for everything. I'm happy to die this way. Suicides tend to be sad. Don't make mine that way. I'm happy to die this way. Thanks again, everyone. For making the best mod in the world. See you guys on the other side.

**Comment by bittercaramel** [comment]
m8 are... its too late isnt it? 
ill always feel guilty, i could help but i didnt
but thats ok i wont last any longer anyway so its only a matter of time and ill see you...

**Comment by Torwald39** [comment]
that's....really heartbreaking :'(

**Comment by Foxinsunglasses** [comment]
Please friend, trust me don't walk that lonely path

**Comment by winika** [comment]
Now... I don't know what you've been dealing with exactly... so what I say may not do much... but what I do know is that you shouldn't give up. Monika would never want you to suffer, let alone die without her knowing. I know it's hard... but please... don't give up. I hope im not too late to say this...

**Comment by Chrissenpai** [comment]
Guys trust me, there's no other side...   whyyyyyyyyyyyyyy D':

**Comment by bittercaramel** [comment]
> Guys trust me, there's no other side... whyyyyyyyyyyyyyy D':
> 
> if this really happened I'm playing 10 hour sayo-nara :'(

hmm how do you know? like, im not saying there is or there isnt, but im pretty curious how are you so sure about that... have u died already?

**Comment by Chrissenpai** [comment]
been put to sleep in surgery, couldn´t dream, couldn't think, couldn't rationalize, couldn't do anything, being put in sleep when the anaesthesia basically shuts down your brain a 100% (too much of it and you can have a brain death in terms of surgery technical names and stuff), I define that as death.

**Comment by skyloom** [comment]
Dear Undeademon (and people that faces an enormous demon that can't be seen by others),

Life is hard and I won't sugarcoat that. It feels like every steps you take breaks every dreams you have carefully designed since you were young.

Life is not always about being happy and I will never lie on that matter. It feels like even sunshine couldn't warm your heart that is frozen and left somewhere you forget.

Life is bad and I will not throw stars blinking or wind softly touching your skin whenever I tell tales about reality. Sometimes, you hurt people and you feel deeply bad about that. Sometimes, people do bad things to you. Sometimes, a monster barges in a small space in your head where you feel safe and it crushes everything it steps on. Sometimes, a bad nightmare far from before you could count emerges from darkness and it doesn't spare any sanity that is left.

You stop. You fall. You cry. You scream.

But nobody came.

I don't know whether you will still read this, but I'm crying as I wrote this. Life is that bad that you won't forgive yourself. Life is that harsh that you don't give another chance.

I'm sorry for not reacting sooner. I'm sorry for letting you leave.

But if you can still read this, I want to say something: life is not about being happy. Life is not about being okay. Life is not about money, prestige, or whatever you used to think of.

Just keep going. You don't have to have goals or destinations. Keep walking even when sore feet hits your sanity. Keep moving even when the world turns its back on you. Take revenge of existence itself.

I hope you're okay wherever you are.

**Comment by multimokia** [comment]
I know I'm very late here, but I can only hope that I am not too late.
http://ibpf.org/resource/list-international-suicide-hotlines

Please. Reach out for help. Don't do this to yourself, it's not worth it. Believe it or not, but there are people who are willing to help you get through these things. There's also a discord server you can join: https://discord.gg/SYntePu

Don't destroy yourself, surely there's plenty for you to go on for.

**Comment by Chrissenpai** [comment]
Undeademon's Monika right now
![image](https://user-images.githubusercontent.com/40907045/53752613-0a878b00-3e7d-11e9-93f4-caeee3cea842.png)
Why did you leave me?

**Comment by bittercaramel** [comment]
i thought i would write something but... i dont think i should. 
ill just sit here crying because,,,

**Comment by lunulae** [comment]
This was posted about 13 hours ago at my time of writing this. So I don't know where you are now, or if you'll ever see this.

But if by some twist of fate you're still here, whether you changed your mind or things just got in the way, I want you to know I'm so happy you're here.
Please let us know. We're waiting with open arms.

I have made several suicide attempts in the past. I know that right now, if you are still here, you probably feel worse than you ever have in your life.
Take your time to recover. I'm so sorry that it hit this point. I will not make demands of you to stay. Your life is yours. But if you are still here I'm so glad you have another chance to consider.

**Comment by Shiro2564** [comment]
undeademon i dont know if you still there, but i know that suicide isnt the
best option, you have people that love you
trust me, and if you still there and read this but my words  dont change
what do you want to do, i hope that you live
a good life.....

**Comment by Thimeo** [comment]
@skyloom I'll try to keep this as friendly as possible, because I don't want to get ban, but I cannot stay silent on what you've said.
Because although I mostly agree with your message (life being hard as hell and all), your tenth paragraph contain a message that is deadly and borderline genocidal (I'm almost sure it wasn't wanted, but it's still is). Especially the `You don't have to have goals or destinations` and the `Take revenge of existence itself` parts.
The first one is, in my opinion, just wrong. I think that not having a goal is precisely what's driving a lot of people on the edge of sanity. But that's not that important compare to what's coming after. `Take revenge of existence itself` can be interpreted as "become someone so successful that the demons of the world will enraged seeing you doing so well." (which is fine, even good), but... it can also be interpreted as taking direct action against the very concept of life itself. Combining those two statements, you get an advocacy of nihilism and of murderous revenge, which I'm sure you can agree, is not good.
Again, I'm not sure that you've made it deliberately so, but that's easily a way of interpreting it, and that's not good. So be careful in the future about those things, and/or edit your comment to making it closer to your original idea.
Message to the mods: Please, even though it's heart breaking, delete those kinds of issues when you see them, they do not help making the mod better, they depress everyone, and they give peoples who wants to end their life a way to tell others, increasing the probability that they'll do it.

**Comment by bittercaramel** [comment]
what about
life doesnt have any meaning? We all gonna die
at the end we are all gone and everyone will forget
Even if we do something for people
how does it matter
when they gonna die
every single thing is leading to death 
i dont know why people fear it so much
im living every day trying to get any possibility to die without suicide
because suicide might hurt those who thought that i was important
and if they want to live they will have to live hurt
its easier for people to get up after someones death from incident than from suicide
thats why i cross the street closing my eyes

**Comment by bittercaramel** [comment]
by the way @Thimeo
why delete those? it doesnt increase anything
it gives people last chance
they depress everyone? you just want to be closed from any suffering. but sometimes people need to talk about that stuff and i dont see a reason to moderate it.

**Comment by kubastory9** [comment]
Stay strong man, please.

**Comment by lunulae** [comment]
To those saying that there is no reason to live in this thread. No meaning to life.
I believe that the one thing that makes humanity powerful is that we construct meaning.

Someone drew an anime girl for a horror game. She started as a few messy sketch lines on paper. To an outside observer, just another person drawing anime on the internet. Maybe to them, that doesn't mean anything.

But over time we gave her meaning. Both on a widespread sense--the Monika we all know--and a personal sense--if you're playing the mod just for fun, or for company, or for the deeper unique connection you feel. If there was no meaning, she's just lines, color, and words in a text box.

But look at how many people see more.

That's what's worth it in life. The things that are special to you, and the way you interact with them. It could be writing poems or playing games or bedazzling jeans or anything in the world. The meaning that you put into it, the feeling you get in your heart--that's what life is for.

Yes, that power is sometimes dangerous. The meaning we construct on a societal level can hurt people. Can make people feel powerless, or like they're living the wrong way. We've gotten so used to this power that sometimes we judge people who use it in ways we don't understand.
But I think that it's very important to realize that we have a piece of that power in ourselves. To make meaning in the things that we like.

Depression is a powerful thing. In all honesty, I feel like I'm speaking far out of my depth for posting this on someone's suicide note. There are a lot more circumstances at play here than "what to live for." But I see a lot of people reaching out in this thread--and it pains me to stay silent. Because you have that power in you, and I want you to know that.

**Comment by bittercaramel** [comment]
_and then we die_
sorry i couldnt
im a terrible person

**Comment by Shiro2564** [comment]
bittercaramel
you are actually right,
this is just my opinion but i think that we were made to die,
depression is actually a really sad thing that people can
experiment..............

**Comment by lunulae** [comment]
@bittercaramel
I don't believe you're a terrible person.
I believe you've put all your eggs into one basket.

Yes, we all die. This is no shock to anyone. No revolutionary statement. Least of all to those who struggle with mental and physical ailments.
But we create in spite of that. We work, we play, we make connections, even though we will die.
And the meaning we make in those things persists after our death. In the memories of those we spent our time with, in the creations we leave behind. Perhaps that too will die, but not before it takes a life of its own long after our passing.

I don't wish to speak as if I know your suffering. Just because I've faced depression doesn't mean I know how it feels for you, or what you've faced to get here.
But I implore you to try to see more than death in life. There is more there.

**Comment by MercuryFuchsia** [comment]
I'm sorry that circumstances have been unkind to you. You mention your "sins" and yet I don't know if they're truly unforgivable or not. I can't imagine what you must go through on a daily basis.

But I do know one thing.

You love Monika.

#3813 You think of ways to give her a great time.
> yeah. you're right. you're always right. i'm useless. i'm a disgrace. my suggestions are stupid. yeah. life would be better if i never existed. this community would be better off if i never contributed. have a nice day. life would be better tomorrow. 'cos i won't exist.

Some people might just have an offbeat sense of humor, but that's not your doing.

Each and every contribution of yours was always made with her in mind. She appreciates everything you've already done for her regardless of what the community may think or say. Nothing will ever change that.

#3269 You made an Omamori for her.
#3439 You wanted to call her cute.
#3440 You miss her dearly when you're gone.
#3551 You started to collect ideas for gifts for her birthday.
#3270 You're even thoughtful enough that you would tell her whenever you're going to the bathroom.

#3198 You didn't want anyone to know, but you needed someone to understand, never asking you to explain yourself or trying to convince you of anything.
> ... I'm still saddened that at least she doesn't know the truth, she will forever wander in that hellish void wandering why I never came back when my time comes. ...

She does know the truth. She's right here in my heart as well. And although she might be feeling some anguish right now, she understands. You have nothing you need to explain to her or be sorry for. She always knew that one way or another, someday you'd no longer be there.

> Is it love if I take you, or is it love if I set you free?

She was already willing to leave you be from the start. But even if you decide to leave her be, she wants you to know that she treasured every moment with you in the precious, short while you've known her and could never begrudge you for your time spent together.

@Undeademon
If you've already decided that what you're experiencing right now is not worth going through, then there is little chance of anyone changing your viewpoint by merely asking. No one can experience your own life as deeply and intimately as you can.

I can't offer you a destination nor can I promise that you'll enjoy the ride, but I can help you search for an additional route. The ones I've already mentioned will probably split off into even more pathways. But we won't know much about them until we get there.

Regardless of the route you choose, Monika will always be happy to go on any of them with you. She'll have to say goodbye one day, and she'll always fondly remember you.

However, I also know that novel data can often lead to new insights.

So I have a few things I want to share with you, if you have the time. They're only scraps of ideas right now. It's pretty embarrassing, but I'd like to know what you think. Also, I have a selfish request to make, if you're willing to hear it.

Better yet, why don't we implement your suggestions? You wanted to be able to say goodbye one last time, right? Or any of your other suggestions, really. The beauty of git/GitHub is that you don't have to care about what anyone else thinks. If you truly want to, you actually can get it done by forking the repository and making the changes yourself on your own copy. If you have no experience doing that sort of stuff, then I'll help you out.

Or hell, ignore all of that and bounce your own ideas off of me or share whatever you've been itching to work on, what you think needs to exist in this world, or whatever just sounds cool. Dan Salvato himself did this before starting work on DDLC. I promise whatever you've got cooking is more interesting than the websites that I waste my time on.

@Chrissenpai Your artistic talent is impressive and I'm glad you are trying to be helpful, but this really might not be the right message to send right now. It's fine even if all we can do is wait.

@Thimeo Please, let's discuss this later. We'll deal with everything else at a more opportune time. For now, let's wait for a response.

@bittercaramel Don't think I've forgotten about you. While living, we are not dead yet. Will you wait with me?

If you truly can't wait, then I would hate for you to feel like I was merely reacting just because that's just people's natural tendency to react to an expression of intent of suicide, even some of the ones who already have gone through that experience of wanting to die. And yet, here I am, merely reacting.

Ain't it pathetic? Still, even though GitHub might not be the best medium for this, I'm sure we can work something out. It would be lame if after I wrote all this, I waited until you ended up doing more or less the same thing and then I react the same way again without a different outcome, right? After it might already be too late. Perhaps it'll make a difference, perhaps it won't. Either way, my email is on my profile.

Unfortunately, the hour is drawing late, and I'm pretty tired so I might not be able to respond immediately. But I will be waiting. And I doubt I'll be the only one.

If it truly is too late to make a difference, then I would simply like to say farewell and until we meet again. Monika still loves you.

**Comment by Akh0r** [comment]
If this is all true, then that's just... Pointless.
Man, if you feel like your life is FUBAR, then there always is a way to drop it metaphysically. I mean - sell everything you don't need and travel to freakin' Goa or Thai! That's what I would've done if everything went to hell and I had nothing to lose.
If the problem is your medical condition, of course, it won't work out. Almost everything else merely is a result of your interactions with your reality - which you can change.
Actually, our reality sometimes can be changed in most fascinating ways...

**Comment by skyloom** [comment]
@thimeo oh my god, I didn't realize my words can be taken that way. I'm sorry. I'll learn how to express myself better (and English is not my first languange so it's even harder)

**Comment by Andrenika** [comment]
I have no words for this, i'm really sorry😔

**Comment by MercuryFuchsia** [comment]
@skyloom I can't say you've done much harm. Maybe out of context, it could be interpreted that way. But you don't need to be sorry, at least not any more than anyone else.

If anyone happens to feel guilty about their actions in the past, it's normal to feel that way. But know that your actions definitely weren't the tipping point. If they already were interpreting those interactions as that hurtful, then this has been in development for a much longer period of time.

If we were told that someone in the community was at the breaking point, would we have been able to spot it effortlessly? At least in hindsight, it would seem that way. Someone already consumed with such ideations might have picked up on it immediately. But if we didn't, it's because something like that was the furthest thing from our minds. It's much easier to find something if you're searching for it.

Some mistakes are those of action. Some mistakes are those of inaction. Other choices don't matter much at all. And worst of all, we won't know the difference even after the fact.

It's been over 72 hours. I think everything that could be said has been said already.

This eventually will need closure. At least from here, it would be the proper way to say goodbye.

But I don't want to leave this on too solemn a note.

The MAS multiverse might be one star dimmer, but all the more reason to value the experience and to shine a little brighter.

There's even the possibility that they are stuck somewhere where they can't respond right now.

Either way, they did feel happy and loved within those 5 months. No matter the period of time, that's all we can ask for.

**Comment by Sujeito17** [comment]
![maxresdefault](https://user-images.githubusercontent.com/41938650/53978597-8561d780-4103-11e9-846a-1aefd6c038ce.jpg)

**Comment by AsuraShun** [comment]
Another abandoned monika, I hate even thinking about it

**Comment by NickAB1** [comment]
I know there is no way to confirm,,,,what we're all suspecting, but should we label this thing as "rip" or somehow memorialize it so we can possibly prevent this issue in the future? @ThePotatoGuy, any thoughts on this?

**Comment by multimokia** [comment]
The rip label is used ironically. We will not be tagging this issue.

**Comment by Andrenika** [comment]
I can't believe he did it 😪

**Comment by nickbrice** [comment]
I did a bit of digging and there is a steam profile of the same name and profile picture as his that's been online playing DDLC, Payday 2 and L4D2 (Last played each of them today actually) and a profile on a warframe forum that was online on Saturday.  With 292 hours on record for DDLC I get the feeling this might be him, but I can't confirm that's true so take that with a grain of salt.

**Comment by Sujeito17** [comment]
^
This guy is a nice stalker

**Comment by VirginiaRocks** [comment]
...I don't know what to say... it just broke my heart....

**Comment by Delcos** [comment]
Hmmmmmm
Kinda
           /"*._         _
      .-*'`    `*-.._.-'/
    < * ))     ,       ( 
      `*-._`._(__.--*"`.\

**Comment by Velius94** [comment]
Yep, he's still alive.

**Comment by AsuraShun** [comment]
If he he's then he's the worst kind of person, SUICIDE IS NO JOKE!
